src/db/car/many.rs

```diff
@@ -206,23 +206,23 @@ impl<WriterT: SettingsStore> SettingsStore for ManyCar<WriterT> {
 }
 
 impl<WriterT: EthMappingsStore> EthMappingsStore for ManyCar<WriterT> {
-    fn read_bin(&self, key: &eth::Hash) -> anyhow::Result<Option<Vec<u8>>> {
+    fn read_bin(&self, key: &eth::types::EthHash) -> anyhow::Result<Option<Vec<u8>>> {
```

Comment:
Could we avoid fully qualified paths? So just use `key: &EthHash`.

Reply:
fixed